bin/ten_of_us.yml

```diff
@@ -999,6 +980,42 @@
       test_location: "local"
     when: config_info.test_exec_location == "local"
 
+- hosts: test_group
+  vars_files:
+    - ansible_vars.yml
+    - upload_files.yml
+  user: root
+  become: true
+  gather_facts: no
+  tasks:
+  - name: restore uploads on bare metal systems if need be.
+    block:
+    - name: See if uploads_back_zathras_exists
+      stat:
+        path: "/{{ config_info.user_parent_home_dir }}/{{ config_info.test_user }}/uploads_back_zathras"
+      register: stat_result
+    - name: restore dir if required
+      block:
+      - name: First remove the uploads we created.
+        command: "rm /{{ config_info.user_parent_home_dir }}/{{ config_info.test_user }}/uploads"
```

Comment:
Use [file module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-state) to remove files.